### DISCUSSION TITLE
Integrating with laravel 10

### DISCUSSION THREAD

**Discussion by abanghendri** [mostafizurhimself/admintoolkit-html#187] [Q&A]
Hi, I am new in vite and tailwind, and I really love your admin dashboard so I try to integrate it with laravel 10, I copy the tailwind.config.js, all images foldee, all js, json and scss to resources folder, copy all dependencies on package.json

I run `npm run dev`, no errors found
I add `@vite(['resources/scss/app.scss', ['resources/js/app.js']`
But when I access my app the looks is really messy
I check the page sources, the css and js pointing to the correct files, is anyone face same problem? Ho to fix it?

Thank you

**Comment by mostafizurhimself** [comment]
Is it possible for you to share an example repository with this issue?

**Reply by Mukti102** [reply]
Can you tell me how to integrate with laravel? I have tried but still confused why it doesn't work

**Comment by hasib-devs** [comment]
Maybe you are missing any css file or the bundler not configured correctly.
If your project is open source, you can share repository link with us.

**Comment by reasecret** [comment]
I'm testing template with Laravel 10 on localhost and did this changes:
Changed template's css filename to panel.css because I'm using different template for frontend view.
Imported this file on panel.js:
```
import '../css/panel.css';
import './panel/app';
import './panel/apexcharts.common';
// And other js files
```
vite.config.js file:
```
input: [
                'resources/css/app.css',
                'resources/css/panel.css',
                'resources/js/app.js',
                'resources/js/panel.js',
            ],
```
On my panel layout, I use like this:
`@vite(['resources/js/panel.js'])`

And this is my admin view. Looks like no problem for now.
![resim](https://github.com/mostafizurhimself/admintoolkit-html/assets/32717963/f29fbc41-8ce5-4e7a-8a70-b5770f984b2b)

Maybe you didn't import template's js files.
